src/lime/tools/AIRHelper.hx

```diff
@@ -199,14 +210,36 @@ class AIRHelper
 		if (targetPlatform == ANDROID)
 		{
 			Sys.putEnv("AIR_NOANDROIDFLAIR", "true");
+
+            if(project.config.getString("air.android_package", "apk") == "aab"){		// build aab
+                args.push("-platformsdk");
+                args.push(project.defines.get("ANDROID_SDK"));
+
+                System.runCommand(workingDirectory, project.defines.get("AIR_SDK") + "/bin/adt", args);
+            } else {	// build apk x32 and/or x64
+                if(hasARM64)
+                    System.runCommand(workingDirectory, project.defines.get("AIR_SDK") + "/bin/adt", args);
+
+                if(hasARMV7 || project.architectures.length == 0){
```

Comment:
This is redundant. `hasARMV7` was already defined as `ArrayTools.containsValue(project.architectures, Architecture.ARMV7) || project.architectures.length == 0`.